undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemSchema.java

```diff
@@ -66,10 +66,11 @@ public enum UndertowSubsystemSchema implements PersistentSubsystemSchema<Underto
     VERSION_13_0(13),   // WildFly 27       N.B. There were no schema changes between 12.0 and 13.0!
     VERSION_14_0(14),   // WildFly 28-32
```

Comment:
While editing this enum please fix this one: this is actually wrong, this should really be 28-present since this is still the default in the `Stabilitity.DEFAULT`.